packages/file/src/touch.coffee.md

```diff
@@ -60,18 +59,17 @@ require('nikita')
         exists = await @fs.base.exists target: config.target
         log message: "Destination does not exists", level: 'INFO', module: 'nikita/lib/file/touch' if not exists
         !exists
-      unless status
-        @execute
-          cmd: "touch #{config.target}"
-          shy: true
-      log message: "Access and modification times updated", level: 'DEBUG', module: 'nikita/lib/file/touch'
       if status
         @file
           content: ''
           target: config.target
           mode: config.mode
           uid: config.uid
           gid: config.gid
+      else
+        @execute
+          cmd: "touch #{config.target}"
+          shy: true
```

Comment:
If not already there in the documentation, could you add a section explaining the behavior of status in this implemtation, it will only be true if the file is created or changed but not if the atime/mtime (or whatever touch does).

Reply:
Also, please add a todo section, we are not checking uid/gid/mode if the file already exists, executing `touch` is just a quick fix because I was to lazy (or lack the time) to implement touch correction with the fs actions.